### ISSUE TITLE
Renovate bot is no longer automatically opening depedency update PRs

### ISSUE THREAD

**Issue by mpeddada1** [googleapis/google-http-java-client#1964]
* Checking update checkboxes from [Dependency Dashboard](https://github.com/googleapis/google-http-java-client/issues/1290) makes no difference.
* Previous update to native-image-shared-config had to be made manually (See https://github.com/googleapis/google-http-java-client/pull/1961). 
* The last renovate bot update to native-image-shared-config was on May 2 (https://github.com/googleapis/google-http-java-client/pull/1937). The latest renovate bot update for other dependencies was on May 31st (https://github.com/googleapis/google-http-java-client/pull/1954)

**Comment by mpeddada1** [comment]
From https://github.com/googleapis/google-http-java-client/issues/845:
```
Location: config
Error type: Invalid allowedVersions
Message: The following allowedVersions does not parse as a valid version or range: "[32.0.0-android,)"
```

**Comment by mpeddada1** [comment]
Verifying with the same configs as https://github.com/googleapis/google-http-java-client/blob/main/renovate.json in https://github.com/mpeddada1/renovate-bot-test/issues/13 doesn't reproduce the same error.

**Comment by ldetmer** [comment]
this seems to have been fixed with #1969 and I see that renovate-bot is opening PRs again #2018
